extensions/docker-desktop/apps/clustermgr/pkg/kubeconfig/internal/kubeconfig/main_test.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2022 VMware Tanzu Community Edition contributors. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package kubeconfig
+
+import (
+	"io/ioutil"
```

Comment:
`ioutil` should not be used as it was deprecated in go 1.16.

Reply:
🤦 Fixing it right-away.